src/main/java/me/botsko/prism/database/sql/SQLPrismDataSourceUpdater.java

```diff
@@ -35,8 +42,7 @@ public void v5_to_v6() {
         String prefix = dataSource.getPrefix();
         String query;
         try (Connection conn = dataSource.getConnection();
-             Statement st = conn.createStatement())
-        {
+             Statement st = conn.createStatement()) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['try resource' has incorrect indentation level 13, expected level should be 14.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5438808)